src/Entity/PageManager.php

```diff
@@ -35,10 +36,21 @@ final class PageManager extends BaseEntityManager implements PageManagerInterfac
     public function __construct(
         string $class,
         ManagerRegistry $registry,
-        private SlugifyInterface $slugify,
+        private SlugifyInterface|SluggerInterface $slugger, // NEXT_MAJOR: Reduce $slugger to SluggerInterface
         private array $defaults = [],
         private array $pageDefaults = [],
     ) {
+        if ($slugger instanceof SlugifyInterface) {
+            trigger_deprecation(
```

Comment:
let's also add a requirement for `symfony/deprecation-contracts` then to composer.json.

Reply:
heheheh true 😄